downloads/index.md

```diff
@@ -172,7 +172,7 @@ Checksums for this release are available in both, [MD5](https://julialang-s3.jul
     <tr>
       <th> Generic Linux on x86 <a href="/downloads/platform/#linux_and_freebsd">[help]</a></th>
       <td colspan="3">
-        <a href="https://julialang-s3.julialang.org/bin/linux/x64/{{upcoming_release_short}}/julia-{{upcoming_release}}-linux-x86_64.tar.gz">64-bit (glibc)</a>
+        <a href="https://julialang-s3.julialang.org/bin/linux/x64/{{upcoming_release_short}}/julia-{{upcoming_release}}-linux-x86_64.tar.gz">64-bit (glibc*)</a>
```

Comment:
I would put the asterisk on musl rather than glibc, since that's the one with the caveat.